### DISCUSSION TITLE
WORKAROUND: Fedora 34 - Podman - WRONGPASS invalid username-password pair or user is disabled.

### DISCUSSION THREAD

**Discussion by chrisgsmith40** [netbox-community/netbox-docker#647] [General]
Hey now, sharing with the world so no one else looses time - 

If you attempt to spin up these netbox-docker containers via ONLY podman-compose, you may encounter something about admin challenge being a WRONGPASS.  Someone who knows Fedora 34 WAAAAAY better than I can investigate properly (except for an early "Warning: Could not create server TCP listening socket *:6379: bind: Address in use"-type 'redis' container start message, I couldn't figure it out by the obviously places, like Firewall pokes and/or SELINUX, or was that Warning related or just a red-herring, etc....)

The articles at the end were extremely helpful, with the nut of it being ...

> sudo dnf -y install podman podman-docker docker-compose git

... forcing the Fedora 34 install to "translate" docker-compose commands.

They are not completely unreadable, but have also attached a text copy of my build notes - mostly for prosperity.

Regards,
Christopher
<NOTES>
[https://fedoramagazine.org/use-docker-compose-with-podman-to-orchestrate-containers-on-fedora/](url)
[https://marcusnoble.co.uk/2021-09-01-migrating-from-docker-to-podman/](url)

[NetBox-build-20211107a.txt](https://github.com/netbox-community/netbox-docker/files/7491554/NetBox-build-20211107a.txt)

**Comment by saman-com** [comment]
did you mange to install the netbox with rootless podman-compose ?

**Reply by chrisgsmith40** [reply]
… not without also installing via dnf docker-compose. Inside my notes you should see a line like …

sudo dnf -y install podman podman-docker docker-compose git
